corehq/ex-submodules/pillowtop/processors/elastic.py

```diff
@@ -90,6 +90,8 @@ def send_to_elasticsearch(index, doc_type, doc_id, es_getter, name, data=None, r
 
     """
     data = data if data is not None else {}
+    del data['_id']
```

Comment:
can you explain this change

Reply:
@snopoke in the new version of ES field ```_id``` is reserved by ES and we can't send in the doc but only as a parameter